package.json

```diff
@@ -112,7 +112,7 @@
         ],
         "clearMocks": true,
         "transformIgnorePatterns": [
-            "node_modules/(?!(@ckeditor|ckeditor5|array-move|lodash-es)/)"
+            "node_modules/(?!(@ckeditor|ckeditor5|array-move|lodash-es|vanilla-colorful)/)"
```

Comment:
required by ckeditor as it requires modern import statements we need to babelify it for jest runtime